### PR TITLE
Fixes typo in idlelib/idle_test/example_stub.pyi

### PR DESCRIPTION
**Pull request by jevandezande**
Fixes typo

### CODE REVIEW

File: Lib/idlelib/idle_test/example_stub.pyi [hunk 1]
```diff
@@ -1,4 +1,4 @@
-" Example to test recognition of .pyi file as Python source code.
+"""Example to test recognition of .pyi file as Python source code."""
```
Comment (Wulian233) [inline Lib/idlelib/idle_test/example_stub.pyi]:
```suggestion
"""An example to test the recognition of .pyi files as Python source code"""
```
LGTM

**Comment by picnixz** [comment]
This might be intentional to have an incomplete file like this.

**Comment by Wulian233** [comment]
https://github.com/python/cpython/pull/113222
This might be a bug

File: Lib/idlelib/idle_test/example_stub.pyi (hunk 1 above)
Comment (terryjreedy) [reply Lib/idlelib/idle_test/example_stub.pyi]: Wulian is correct below that `"` should have been `#`.  I will resolve this and make a better fix.

Comment (terryjreedy) [inline Lib/idlelib/idle_test/example_stub.pyi]:
```suggestion
# An example file to test recognition of a .pyi file as Python source code.
```

**Comment by terryjreedy** [comment]
Will merge after 3.13.0.

**Comment by terryjreedy** [comment]
Decided instead to remove 3.13 backport label, merge main and 3.12 now, and do 3.13 backport when possible.  I need to see what happens to status when merge.

**Pull request merged**